jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/projects/jakarta-sample/src/main/java/io/openliberty/sample/jakarta/jsonb/JsonbTransientDiagnostic.java

```diff
@@ -25,6 +25,16 @@
     @JsonbTransient
     private int id;
 
-    @JsonbPropery("fav_lang")
+    @JsonbProperty("fav_lang")
     private String favoriteLanguage;
+    
+    @JsonbProperty("person-id")
+    private int getId() {
+        return id;
+    }
+    
+    @JsonbCreator
+    private int setId(int id) {
+        this.id = id;
+    }
```

Comment:
Consider comments in this class indicating which lines are invalid/diagnostics are expected on

Reply:
Added